nextflow.config

```diff
@@ -336,7 +336,7 @@ manifest {
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
     version         = '3.4dev'
-    doi             = '10.12688/f1000research.16665.2, 10.5281/zenodo.4063683'
+    doi             = '10.12688/f1000research.16665.2, 10.1101/2023.07.19.549462, 10.5281/zenodo.3476425'
```

Comment:
```suggestion
    doi             = '10.12688/f1000research.16665.2, 10.1101/2023.07.19.549462v2, 10.5281/zenodo.3476425'
```
This would point to the more recent bioRxiv version

Reply:
duh! Thank you!. Will also update the readme

Reply:
It seems to me that the `v2` is not part of the doi.

Reply:
you are right, the link goes to 404